File: audio/mp3asm/files/patch-src_utils.c
--- src/utils.c.orig	2001-08-17 20:44:09 UTC
+++ src/utils.c
@@ -25,6 +25,7 @@
 #include "utils.h"
 #include <unistd.h>
 #include <stdio.h>
+#include <errno.h>
 
 /*
  * tmalloc: mallocs cleanly

File: audio/spotifyd/files/patch-src_utils.rs
--- src/utils.rs.orig	2021-02-09 22:43:15 UTC
+++ src/utils.rs
@@ -3,7 +3,7 @@
 #[cfg(target_os = "macos")]
 use whoami;
 
-#[cfg(target_os = "linux")]
+#[cfg(any(target_os = "freebsd", target_os = "linux"))]
 fn get_shell_ffi() -> Option<String> {
     use libc::{geteuid, getpwuid_r};
     use std::{ffi::CStr, mem, ptr};
